setupGitFilters.ps1

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env powershell
+# Setup script for Git filter drivers used in ModularSensors
+# This configures the smudgePasswords and disableDebug filters referenced in .gitattributes
+
+Write-Host "Setting up Git filter drivers for ModularSensors..." -ForegroundColor Green
+
+# Configure the smudgePasswords filter for .ino files
+Write-Host "Configuring smudgePasswords filter..." -ForegroundColor Yellow
```

PSScriptAnalyzer (1.24.0)

[warning] File 'setupGitFilters.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)